style.css

```diff
@@ -0,0 +1,551 @@
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  padding-left: 24px;
+  padding-right: 24px;
+  margin-top: 10px;
+  margin-bottom: 100px;
+  flex-direction: row;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.headline {
+  background-image: url("./image/headlinebackground.png");
+  height: 60vh;
+  padding-left: 24px;
+  padding-bottom: 100px;
+  border-bottom-left-radius: 70px;
+  background-color: #fff;
+}
+
+.Primay-text {
+  flex-direction: row;
+  margin-bottom: 12px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 1000;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.secondary-text {
+  display: flex;
+  flex-direction: row;
+  height: 146px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  margin-top: 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.social-links {
+  display: flex;
+  gap: 1rem;
+  padding-left: 0;
+  align-items: baseline;
+  flex-direction: row;
+}
+
+.connect {
+  height: 24px;
+  display: flex;
+  flex-direction: row;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.work1-image {
+  display: flex;
+  flex-direction: row;
+  height: 220px;
+  background: url(blog_drbl_4x.png);
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.work1-title {
+  display: flex;
+  flex-direction: row;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 12px;
+}
+
+#p1 {
+  display: flex;
+  flex-direction: row;
+  height: 16px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-left: 16px;
+}
+
+#p2 {
+  display: flex;
+  flex-direction: row;
+  height: 16px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.project-container {
```

Comment:
- [ ] great job. Kindly ensure the main container of the projects has a display:grid. it will help to align and balance your designs, and comes in handy when you want maintainable webpages.